src/lib/index.js

```
@@ -2,84 +2,84 @@
-      o < 3 ? e : r === null ? (r = Object.getOwnPropertyDescriptor(e, t)) : r,
+      n < 3 ? e : o === null ? (o = Object.getOwnPropertyDescriptor(e, t)) : o,
     s;
   if (typeof Reflect == "object" && typeof Reflect.decorate == "function")
-    i = Reflect.decorate(n, e, t, r);
+    i = Reflect.decorate(r, e, t, o);
   else
-    for (var a = n.length - 1; a >= 0; a--)
-      (s = n[a]) && (i = (o < 3 ? s(i) : o > 3 ? s(e, t, i) : s(e, t)) || i);
-  return o > 3 && i && Object.defineProperty(e, t, i), i;
+    for (var a = r.length - 1; a >= 0; a--)
+      (s = r[a]) && (i = (n < 3 ? s(i) : n > 3 ? s(e, t, i) : s(e, t)) || i);
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)